src/version/mod.rs

```diff
@@ -128,11 +128,10 @@ impl Version {
     }
 
     pub fn record_readme_rendering(&self, conn: &PgConnection) -> QueryResult<usize> {
-        use schema::versions::dsl::readme_rendered_at;
-        use diesel::dsl::now;
+        use schema::readme_renderings::dsl::*;
 
-        diesel::update(self)
-            .set(readme_rendered_at.eq(now.nullable()))
+        diesel::insert_into(readme_renderings)
```

Comment:
Doesn't this need to be an upsert? Or does `insert_into` handle upserts automatically? Occasionally, when we change the readme rendering, I rerender all the readmes, which should update this timestamp.

Reply:
Good catch